src/sources/file.rs

```diff
@@ -142,15 +150,19 @@ pub struct FileConfig {
     ///
     /// Off by default, the offset is only added to the event if this is set.
     #[serde(default)]
+    #[configurable(metadata(docs::examples = "offset"))]
     pub offset_key: Option<OptionalValuePath>,
 
-    /// Delay between file discovery calls, in milliseconds.
+    /// Delay between file discovery calls.
     ///
-    /// This controls the interval at which files are searched. A higher value results in greater chances of some short-lived files being missed between searches, but a lower value increases the performance impact of file discovery.
+    /// This controls the interval at which files are searched. A higher value results in greater
+    /// chances of some short-lived files being missed between searches, but a lower value increases
+    /// the performance impact of file discovery.
     #[serde(
         alias = "glob_minimum_cooldown",
         default = "default_glob_minimum_cooldown_ms"
     )]
+    #[configurable(metadata(docs::type_unit = "milliseconds"))]
     pub glob_minimum_cooldown_ms: u64,
```

Comment:
I think we want to turn this into a Duration with the appropriate serde_as calls to parse the units correctly.

Reply:
I swear there was something that didn't work properly here, but I think I just got it lumped in with the `Option`d ones 🤷 - fixing

Reply:
Addressed in [0e7c38b](https://github.com/vectordotdev/vector/pull/16099/commits/0e7c38bf4f326cfe4575ba418a6da0996b4a8658)[0e7c38b](https://github.com/vectordotdev/vector/pull/16099/commits/0e7c38bf4f326cfe4575ba418a6da0996b4a8658)